packages/gafl-webapp-service/src/handlers/page-handler.js

```diff
@@ -132,7 +132,6 @@ export default (path, view, completion, getData) => ({
     pageData.analyticsMessageDisplayed = retrieveKeyOrFalse(analytics, ANALYTICS.seenMessage)
     pageData.analyticsSelected = retrieveKeyOrFalse(analytics, ANALYTICS.selected)
     pageData.acceptedTracking = retrieveKeyOrFalse(analytics, ANALYTICS.acceptTracking)
-    pageData.gtmContainerId = process.env.GTM_CONTAINER_ID || false
```

Comment:
We still need this value as it's included in the GTM script on the page (see lines 175 and 177 of the njk file)